### PR TITLE
Adding adamdehaven's custom userspace(s) for kbdfans/kbd75 and kbdfan…

### PR DESCRIPTION
**Pull request by adamdehaven**
Adds custom userspace under `/keyboards/kbdfans/kbd75` and `/keyboards/kbdfans/kbd67`

## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by adamdehaven** [comment]
Also removes duplicate `#define RGBLIGHT_SLEEP` line in `/keyboards/kbdfans/kbd75/config.h`

File: keyboards/kbdfans/kbd67/mkii_soldered/keymaps/adamdehaven/rules.mk [hunk 1]
```diff
@@ -0,0 +1,3 @@
+DEFAULT_FOLDER = kbdfans/kbd67/mkii_soldered
+
```
Comment (zvecr) [inline keyboards/kbdfans/kbd67/mkii_soldered/keymaps/adamdehaven/rules.mk:2]:
```suggestion
```

File: keyboards/kbdfans/kbd75/config.h [hunk 2]
```diff
@@ -42,7 +42,6 @@
 #define RGBLIGHT_HUE_STEP 8
 #define RGBLIGHT_SAT_STEP 8
 #define RGBLIGHT_VAL_STEP 8
```
Comment (zvecr) [inline keyboards/kbdfans/kbd75/config.h:44]:
```suggestion
#define RGBLIGHT_VAL_STEP 8
#define RGBLIGHT_SLEEP
```

File: keyboards/kbdfans/kbd75/keymaps/adamdehaven/config.h [hunk 3]
```diff
@@ -0,0 +1 @@
+#define TAPPING_TERM 200
```
Comment (zvecr) [inline keyboards/kbdfans/kbd75/keymaps/adamdehaven/config.h:1]:
```suggestion
#pragma once

#define TAPPING_TERM 200
#undef RGBLIGHT_SLEEP
```

File: keyboards/kbdfans/kbd75/keymaps/adamdehaven/rules.mk [hunk 4]
```diff
@@ -0,0 +1,3 @@
+DEFAULT_FOLDER = kbdfans/kbd75/rev2
+
```
Comment (zvecr) [inline keyboards/kbdfans/kbd75/keymaps/adamdehaven/rules.mk:2]:
```suggestion
```

File: keyboards/kbdfans/kbd67/mkii_soldered/keymaps/adamdehaven/config.h [hunk 5]
```diff
@@ -0,0 +1 @@
+#define TAPPING_TERM 150
```
Comment (zvecr) [inline keyboards/kbdfans/kbd67/mkii_soldered/keymaps/adamdehaven/config.h:1]:
```suggestion
#pragma once

#define TAPPING_TERM 150
```

File: keyboards/kbdfans/kbd75/keymaps/adamdehaven/keymap.c [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+            if (!record->event.pressed) {
+                // on keyup
+                RGB_current_val = rgblight_config.val;
+            }
+            break;
+    }
+    return true;
+};
+
+void matrix_scan_user(void) {}
```
Comment (fauxpark) [inline keyboards/kbdfans/kbd75/keymaps/adamdehaven/keymap.c:94]:
```suggestion
```

File: keyboards/kbdfans/kbd75/keymaps/adamdehaven/keymap.c [hunk 7]
```diff
@@ -0,0 +1,193 @@
...
+     * │  KC_LCTL  │  KC_LGUI  │  KC_LALT  │                            KC_SPC                             │ KC_RALT │MO(_FN1) │ KC_RCTL │ KC_LEFT │ KC_DOWN │ KC_RGHT │
+     * └───────────┴───────────┴───────────┴───────────────────────────────────────────────────────────────┴─────────┴─────────┴─────────┴─────────┴─────────┴─────────┘
+     */
+    [_QWERTY] = LAYOUT( \
+        KC_ESC, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12, KC_PSCR, KC_INSERT, KC_DEL, \
+        KC_GRV, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINS, KC_EQL, KC_DEL, KC_BSPC, KC_HOME, \
+        KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_LBRC, KC_RBRC, KC_BSLS, KC_END, \
+        KC_CAPS, KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT, KC_ENT, TD(TD_PGUP), \
+        KC_LSFT, _______, KC_Z, KC_X, KC_C, KC_V, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, KC_RSFT, KC_UP, TD(TD_PGDN), \
+        KC_LCTL, KC_LGUI, KC_LALT, KC_SPC, KC_SPC, KC_SPC, KC_RALT, MO(_FN1), KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT \
```
Comment (fauxpark) [inline keyboards/kbdfans/kbd75/keymaps/adamdehaven/keymap.c:123]:
```suggestion
    [_QWERTY] = LAYOUT(
        KC_ESC, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12, KC_PSCR, KC_INSERT, KC_DEL,
        KC_GRV, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINS, KC_EQL, KC_DEL, KC_BSPC, KC_HOME,
        KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_LBRC, KC_RBRC, KC_BSLS, KC_END,
        KC_CAPS, KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT, KC_ENT, TD(TD_PGUP),
        KC_LSFT, _______, KC_Z, KC_X, KC_C, KC_V, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, KC_RSFT, KC_UP, TD(TD_PGDN),
        KC_LCTL, KC_LGUI, KC_LALT, KC_SPC, KC_SPC, KC_SPC, KC_RALT, MO(_FN1), KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT
```

File: keyboards/kbdfans/kbd75/keymaps/adamdehaven/keymap.c [hunk 8]
```diff
@@ -0,0 +1,193 @@
...
+     * └───────────┴───────────┴───────────┴───────────────────────────────────────────────────────────────┴─────────┴─────────┴─────────┴─────────┴─────────┴─────────┘
+     */
+    [_FN1] = LAYOUT( \
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_MUTE, \
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+        _______, RGB_TOG, RGB_MOD, RGB_HUI, RGB_HUD, RGB_SAI, RGB_SAD, RGB_VAI, RGB_VAD, _______, _______, _______, _______, _______, _______, \
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_VOLU, \
+        _______, _______, _______, _______, BL_DEC, BL_TOGG, BL_INC, BL_STEP, _______, _______, _______, _______, _______, KC_MPLY, KC_VOLD, \
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_MPRV, KC_MPLY, KC_MNXT \
```
Comment (fauxpark) [inline keyboards/kbdfans/kbd75/keymaps/adamdehaven/keymap.c:147]:
```suggestion
    [_FN1] = LAYOUT(
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_MUTE,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, RGB_TOG, RGB_MOD, RGB_HUI, RGB_HUD, RGB_SAI, RGB_SAD, RGB_VAI, RGB_VAD, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_VOLU,
        _______, _______, _______, _______, BL_DEC, BL_TOGG, BL_INC, BL_STEP, _______, _______, _______, _______, _______, KC_MPLY, KC_VOLD,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_MPRV, KC_MPLY, KC_MNXT
```

File: keyboards/kbdfans/kbd75/keymaps/adamdehaven/keymap.c [hunk 9]
```diff
@@ -0,0 +1,193 @@
...
+    } else {
+    }
+
+    if (usb_led & (1 << USB_LED_COMPOSE)) {
+    } else {
+    }
+
+    if (usb_led & (1 << USB_LED_KANA)) {
+    } else {
+    }
```
Comment (fauxpark) [inline keyboards/kbdfans/kbd75/keymaps/adamdehaven/keymap.c:192]:
```suggestion
    if (usb_led & (1 << USB_LED_CAPS_LOCK)) {
        toggle_rgb_caps_lock(true);
    } else {
        toggle_rgb_caps_lock(false);
    }
```

**Comment by adamdehaven** [comment]
Looks good

**Pull request closed without merging**